stl/src/locale0.cpp

```diff
@@ -1,6 +1,7 @@
 // Copyright (c) Microsoft Corporation.
 // SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 
+// class locale basic member functions

```

Comment:
This was accidentally deleted by me and pointed out by @AlexGuteniev

Is this not just a shouty banner in disguise?